app/controller/DataController.php

```diff
@@ -406,13 +444,24 @@ function metadata() {
         if(!isset($data['og_title'])){
              $data['og_title'] =  $dom->getElementsByTagName('title')->item(0)->textContent;
         }
-        if(!isset($data['og_description'])){
+        if(!isset($data['og_description']) && isset($data['alt_description']) && !empty($data['alt_description'])){
              $data['og_description'] =  $data['alt_description'];
         }
+        
         if(!isset($data['og_img']))
         {
-            $base =  $dom->getElementsByTagName('base')->item(0)->attributes->getNamedItem("href")->value;
-            $data['og_img'] = $base.$dom->getElementsByTagName('img')->item(0)->attributes->getNamedItem("src")->value;
+            $base=$dom->getElementsByTagName('base');
+            if($base->length>0)
+                $base =  $dom->getElementsByTagName('base')->item(0)->attributes->getNamedItem("href")->value;
+            else
+                $base=$url;
+            
+            $imgSrc=$dom->getElementsByTagName('img')->item(0)->attributes->getNamedItem("src")->value;
+            
+            if(substr($imgSrc, 0, 4)=="http")
```

Comment:
Ist in dem Fall auch gecovert, hier ging es um das unterscheiden ob relativ oder absolute Pfade. Allerdings werden https Bilder vmtl auf dmdn nicht angezeigt, aber aus anderen gründen